wcivf/apps/elections/models.py

```diff
@@ -49,7 +49,7 @@ class Election(models.Model):
     objects = ElectionManager()
 
     class Meta:
-        ordering = ["election_date"]
+        ordering = ["election_date", "election_weight"]
```

Comment:
I _think_ this requires a migration change - so you'll want to run `manage.py makemigrations elections --name=update_ordering` and commit the generated file